src/api/hooks/useDuplicateRegistrationSearch.ts

```diff
@@ -25,18 +26,18 @@ export const useDuplicateRegistrationSearch = ({
   const duplicateRegistration = !title
     ? undefined
     : registrationsWithSimilarName.find((reg) => {
-        const isSameRegistration = reg.identifier === identifier;
-        const hasSameName = reg.entityDescription?.mainTitle.toLowerCase() === title.toLowerCase();
+        const isSameRegistration = getIdentifierFromId(reg.id) === identifier;
```

Comment:
Keep it simple. Ingen grunn til å gå gjennom en annen funksjon får å få tak i noe man allerede har

```suggestion
        const isSameRegistration = reg.identifier === identifier;
```